README.md

```diff
@@ -13,6 +13,8 @@ a text editor, eventually...
 
 `pip install babi`
 
+babi works best in a [virtualenv](https://github.com/asottile/babi/issues/234#issuecomment-1095372143)
```

Comment:
I would leave out the link personally

Reply:
Done

Reply:
Anthony, I have one more question related to the the babi on macos.
In the description  I saw special setting "opt as alt/meta" in Terminal.app. But it doesn't work for me. Do you have the same issue?